diff -ruN orig/drivers/gpio/gpio-it87.c save/drivers/gpio/gpio-it87.c
--- orig/drivers/gpio/gpio-it87.c	2019-02-26 19:35:22.356068908 -0600
+++ save/drivers/gpio/gpio-it87.c	2019-02-26 19:29:24.608448674 -0600
@@ -36,6 +36,7 @@
 #define NO_DEV_ID	0xffff
 #define IT8620_ID	0x8620
 #define IT8628_ID	0x8628
+#define IT8721_ID	0x8721
 #define IT8728_ID	0x8728
 #define IT8732_ID	0x8732
 #define IT8761_ID	0x8761
@@ -312,6 +313,7 @@
 		it87_gpio->simple_size = 0;
 		it87_gpio->chip.ngpio = 64;
 		break;
+	case IT8721_ID:
 	case IT8728_ID:
 	case IT8732_ID:
 		gpio_ba_reg = 0x62;
